kin-sdk-core/src/main/java/kin/sdk/core/Account.java

```diff
@@ -0,0 +1,21 @@
+package kin.sdk.core;
+
+
+class Account {
```

Comment:
please add javadoc documentation - what is Account encryptedSeed, accountID etc.